docs/source/frameworks/xgboost.rst

```diff
@@ -2,49 +2,178 @@
 XGBoost
 =======
 
-Users can now use XGBoost with BentoML with the following API: :code:`save_model`, :code:`load_model`, and :code:`get` as follows:
+This guide provides an overview of using `XGBoost <https://xgboost.readthedocs.io/en/stable/>`_ with BentoML.
```

Comment:
```suggestion
XGBoost is an optimized distributed gradient boosting library designed to be highly efficient, flexible and portable.. This guide provides an overview of using `XGBoost <https://xgboost.readthedocs.io/en/stable/>`_ with BentoML.
```

Reply:
FWIW I don't think we need to describe each framework, but it's probably better to be consistent for now.